src/components/Courses/components/CourseCard/CourseCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './CourseCard.css';
+import { Course, Author } from '../../../../types';
+import { formatDuration } from '../../../../helpers/getCourseDuration';
+import { formatCreationDate } from '../../../../helpers/formatCreationDate';
+import DeleteIcon from '../../../../assets/DeleteIcon.svg';
+import EditIcon from '../../../../assets/EditIcon.svg';
+import { deleteCourse } from '../../../../store/courses/actions';
+import { useDispatch } from 'react-redux';
+import Button from '../../../../common/Button/Button';
+
+type CourseCardProps = {
+	course: Course;
+	authors: Author[];
+};
+
+const CourseCard: React.FC<CourseCardProps> = ({ course, authors }) => {
+	const navigate = useNavigate();
+	const dispatch = useDispatch();
+	const getAuthorsNames = (authorId: string[]): string => {
+		if (!authors) return '';
+
+		return authorId
+			.map((id) => authors.find((author) => author.id === id)?.name)
+			.join(', ');
+	};
+
+	const handleDelete = () => {
+		dispatch(deleteCourse(course.id));
+	};
+
+	return (
+		<Link to={`/course/${course.id}`} className='course-card-link'>
```

Comment:
Please wrap title with the link because this way icons on the card aren't clickable